test/FunctionalTests/Tests/DataServices/UnitTests/DesignT4UnitTests/ODataT4CodeGeneratorTestDescriptors.cs

```diff
@@ -196,6 +196,16 @@ private static string LoadContentFromBaseline(string key)
             Verify = (code, isCSharp, useDSC) => VerifyGeneratedCode(code, Simple.ExpectedResults, isCSharp, useDSC, "Simple"),
         };
         #endregion
+        #region SimpleMultipleFiles
+        public static string EdmxSimpleMultipleFiles = LoadContentFromBaseline("SimpleMultipleFiles.xml");
+        public static string SimpleMultipleFilesCSharp = LoadContentFromBaseline("SimpleMultipleMain.cs");
+        public static ODataT4CodeGeneratorTestsDescriptor SimpleMultipleFiles = new ODataT4CodeGeneratorTestsDescriptor()
+        {
+            Metadata = EdmxSimpleMultipleFiles,
+            ExpectedResults = new Dictionary<string, string>() { { ExpectedCSharp, SimpleMultipleFilesCSharp }, { ExpectedCSharpUseDSC, SimpleCSharpUseDSC }, { ExpectedVB, SimpleVB }, { ExpectedVBUseDSC, SimpleVBUseDSC } },
+            Verify = (code, isCSharp, useDSC) => VerifyGeneratedCode(code, SimpleMultipleFiles.ExpectedResults, isCSharp, useDSC, "SimpleMultipleFiles"),

```

Comment:
Does this actual verify that multiple files were generated as expected? It seems to me that it's only verifying `SimpleMultipleMain.cs` was, what about `SimpleMultipleFilesInclude.cs`?

Reply:
Correct. It shows the main file does not contain all the classes. Multiple files in this case is implied by the correct content of the main file.

Reply:
I don't think that correct content of the main file implies that the other files were generated correctly. It may be the case that the other files were truncated, or were mixed up in one file instead of separate file, or some other behaviour. I think it would be worthwhile to also test the other files, and better yet to have at least 2 entity types to at least verify that the files are split into  separate file per class, and each with the correct content.

Reply:
I understand your concerns. Changing the implementation.

Reply:
Fixed

Reply:
I have not been able to spot the tests that verify that multiple files were generated correctly, in this case the `SimpleMultipleFilesInclude.cs` still does not seem to have tests. Also if the test csdl could contain at least 2 entity types in order to demonstrate that the files are split one per type, that would be great in my opinion.